.github/build.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+set -xe
+
+rm -rf bifrost bwa htslib samtools sickle
```

Comment:
Is this line really necessary for the CI?
If not, I would like to avoid 'rm -rf' in case this script is ever executed by a user without paying attention to the working directory and other installations there.

Reply:
I aggree, that this looks like a greate feature for the PopIns2 repo! Thanks a lot, Martin!

Reply:
This line is not really needed for the CI because the CI always starts with a fresh VM. I will remove it!
If a user executes the new version of the script and any of these folders exists then `git clone` will fail with a clear error message.

Reply:
Done!